docs/notebooks/ztf_bts-ngc.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
<p>Can you add some <code>%%time</code> to the beginning of the cells to show which parts are time-intensive? </p><p> </p><p>My gut says that </p><ul><li>those first few cells that download the test files are going to be .. a few seconds</li><li>the process of converting the dataframes into LSDB catalogs is ... another few seconds? </li><li>setting up the crossmatch plan is sub-second</li><li>running the pipeline below is a few seconds?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done</p>